reason-react-native/src/apis/PanResponder.md

```diff
@@ -1,9 +1,137 @@
 ---
-id: apis/Packager
-title: Packager
-wip: true
+id: apis/PanResponder
+title: PanResponder
 ---
 
+`PanResponder` allows more fine-grained control of the _Gesture Responder
```

Comment:
A link to http://facebook.github.io/react-native/docs/gesture-responder-system would be nice